data_cleaning/ice_velocity.ipynb

```diff
@@ -0,0 +1,354 @@
+{
```

Comment:
<p>I think we can remove this. It doesn't add much on binder (for example)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/121/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>